slasher/rpc/update_min_max_span.go

```diff
@@ -0,0 +1,63 @@
+package rpc
+
+import (
+	"github.com/pkg/errors"
+
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+)
+
+// UpdateMaxSpan is used to update the max span of an incoming attestation after the slashing detection phase.
+// logic is following protolambda detection method.
+// from here: https://github.com/protolambda/eth2-surround/blob/master/README.md#min-max-surround
+func (ss *Server) UpdateMaxSpan(source uint64, target uint64, validatorIdx uint64) error {
+	spanMap, err := ss.SlasherDb.ValidatorSpansMap(validatorIdx)
```

Comment:
Can this be called SlasherDB instead of SlasherDb?

Reply:
will add benchmark after the detection pr. sg?